core/dom/element.js

```diff
@@ -2112,6 +2101,55 @@ CKEDITOR.dom.element.clearMarkers = function( database, element, removeFromDatab
 				else if ( !type || node.type == type )
 					callback( node );
 			}
+		},
+
+		/**
+		 * Returns scroll position of the element's document.
+		 *
+		 * @since 4.10.0
+		 * @returns {Object} Object representing vertical and horizontal scroll position of the document.
+		 * @returns {Number} return.scrollTop vertical scroll position.
+		 * @returns {Number} return.scrollLeft horizontal scroll position.
+		 */
+		getDocumentScrollPosition: function() {
```

Comment:
I wonder if this (and `setDocumentScrollPosition`) is longer needed after adding `get/setScrollPosition` to `CKEDITOR.dom.document` itself.